packages/server-wallet/src/__test__/crash-tolerance/crash-tolerance.test.ts

```diff
@@ -125,12 +125,14 @@ it('Create a fake-funded channel between two wallets, of which one crashes midwa
   const resultB2 = await b.updateFundingForChannels([depositByB]);
 
   expect(getChannelResultFor(channelId, resultA2.channelResults)).toMatchObject({
-    status: 'running', // Still opening because turnNum 3 is not supported yet, but is signed by A
+    // Running because turnNum 2 is supported and channel result says 'running' for state > 1
```

Comment:
This doesn't make sense to me. Mind explaining?

Reply:
Turn number 2 is a valid chain from 0, 1, 2 which A has now and the wallet shows `status = "running"` for turn number 2

Reply:
I think that's a bug though?

Reply:
Yeah, but can be fixed in a separate PR.

Reply:
https://github.com/statechannels/statechannels/pull/2794

Reply:
Presumably this will fail when both this pr and #2794 are merged?

Reply:
Rebased.